fastlane/spec/actions_specs/cocoapods_spec.rb

```diff
@@ -96,6 +96,25 @@
 
         expect(result).to eq("cd 'Project' && bundle exec pod install")
       end
+
+      it "adds error_callback to command" do
+        result = Fastlane::FastFile.new.parse("lane :test do
+          cocoapods(
+            error_callback: lambda { |result| puts 'failure' }
+          )
+        end").runner.execute(:test)
+
+        expect(result).to eq("bundle exec pod install")
+      end
+
+      it "error_callback is executed on failure" do
```

Comment:
I can't add the test whether `error_callback` is called.
Because external commands are not executed in testing.

Instead, I added a mock test to check `error_callback` is passed to `sh_control_output`.